docs/deployment-examples/prometheus/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+# Sample Prometheus config
+# This assumes that your Prometheus instance can access this application on localhost:8080
```

Comment:
When you say `this application`, do you mean the Platform? Can you update the wording to be a little bit clearer?